constantine/eth_verkle_ipa/common_utils.nim

```diff
@@ -30,10 +25,12 @@ import
 # ############################################################
 
 
-func generate_random_points* [EC_P](points: var openArray[EC_P], num_points: uint64)  =
```

Comment:
The use of EC_P as a generic parameter here, but later to mean `ECP_TwEdwards_Prj[Fp[Banderwagon]]` was confusing.

Furthermore, we should use affine coordinates for the CRS.